Command uniform

GoldenCan opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
Added three (3) - now four (4) - new pieces of clothing: The command jumpsuit, command jumpskirt, command cap and command beret.
These are intended to be multi-purpose "generic" command clothes. They can only be obtained from the Head of Personnel's uniform printer. They are intended to be available to give out to people promoted to non-standard command or command-related jobs. They can be used as sets or individual pieces of clothing can be used on their own.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
There are currently no "generic" command clothes.

There are generally two categories of non-standard/irregular command jobs that one might get promoted to in a round.
1. A "head" which does not match one of the roundstart command jobs. (Examples: Chief Service Worker, Head of Greytide, Head of Clowning)
2. A job that primarily works with commanad and the bridge, but which is not a head of a department. (Examples: Bridge Assitant, Bridge Guard, Captains personal body guard, HOP's secretary.)

(NOTE: The clothes are not in any way required to be used like described below. It is up to command in a given round to decide when and how to use them, and which to use.)
The jumpsuit, jumpskirt and command cap are intended primarily for jobs in the second category. The clothes are intended to be clearly command colored, but the baseball cap itself does not signal authority as much as the command beret.
The beret is primarily intended for jobs in the first category, as berets in are associated with authority and status in SS14. The jumpsuit and jumpskirt can also be used for jobs of the first category if so desired.


In the game there are several reason that one might promote (or perhaps even demote) somone to a job in one of those two categories.
(I have listed a two reasons why someone might be promoted/demoted to a "Chief Service Worker" below. They are not the best examples.)

But currently, there are no clothes that clearly identifies anyone as a member of command, without also implying that they are the head of one of the 6 (roundstart) departments or the captain. This means that when anyone is given an "irregular" command position there is no good way to for their clothes to indicate their position.

In conclusion: Adding these clothes would not allow command members to do anything that they could not do before, but it does allow them to add more visual distinction and clarity in a varity of rare but interesting cases.


##Two examples of promotions/demotions.
(You do not really _need_ to read this section - it is not that good.)

I will now list examples of why someone might be appointed to a command job which does not correspond to one of the 7 roundstart/regular command jobs:
1. Due to a bureaucratic or clerical error (i forgot which one was which) there are now 2 Heads of Personnel. They decide to split their tasks and duties between them: One HOP is tasked with managing service and its retitled to the "Chief Service Worker", the other HOP keeps the title of HOP whilst only managing the HOP's duties unrelated to service (given people jobs and managing IDs, keeping an eye on EVA, protecting Ian and acting as the second-in-command if needed).
2. The server is low pop and there is no HOP. A chef with experience with botany asks the captain to become the HOP in order to manage the unmanaged service. The captain agrees that she can not manage the entirety of service on their own, but she believes that giving the chef what is almost AA would be too risky, and as such she does not fully promote the chef to HOP, instead giving the chef access to service jobs and the authority to manage service.

(These are two very similar examples. I wanted to add more but i didn't have the time.)

<!-- ## Technical details -->
<!-- Summary of code changes for easier review. -->

## Media
Showcase of the new clothing (UPDATED september 22, 2024):
![ZCommandOutfitUpdated](https://github.com/user-attachments/assets/53f84c86-189d-4fe6-b2f4-dcb33aa9c2b6)
![ZCommandOutfitUpdated2](https://github.com/user-attachments/assets/11116b39-00c8-4874-ad7d-b7b792320d76)


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
- add: Heads of Personnel can now fabricate generic command-themed clothes using their uniform printer.

Catdere commented:
I like the idea but i think the uniform but I think a dark blue turtleneck would look the best and fit the best. 

My reasoning stems from turtlenecks provoking some sort of status that being, The HoS, CE, CMO, RD, Central Command, and Even the syndicate in some places all seem to wear them.

Kadeo64 commented:
i really like the uniform but it doesn't really.. scream "command". In fact, it screams the opposite, this looks like something a low level worker would wear. If the pants/skirt were a dark blue/dark grey maybe it would be better? 

also a turtleneck would work

LordEclipse commented:
These look almost like mail worker to me, maybe turtle neck + generic command pauldrons/cloak would sell the look better.

LittleNorthStar commented:
I think maybe change the hat to a beret as there also common among command and have some level of associated authority in SS14

Kadeo64 commented:
> I think maybe change the hat to a beret as there also common among command and have some level of associated authority in SS14

a beret in this shade of blue with the gold badge would be drippy

GoldenCan commented:
> I like the idea but i think the uniform but I think a dark blue turtleneck would look the best and fit the best.
> 
> My reasoning stems from turtlenecks provoking some sort of status that being, The HoS, CE, CMO, RD, Central Command, and Even the syndicate in some places all seem to wear them.
A turtleneck would show more status. (Although it might not fit in all situations?)

> i really like the uniform but it doesn't really.. scream "command". In fact, it screams the opposite, this looks like something a low level worker would wear. If the pants/skirt were a dark blue/dark grey maybe it would be better?
> 
> also a turtleneck would work
I think it is clearly command-related, but i do agree that it does not show as much status as most of the heads clothing.
(I think not showing as much status/authority but still using command colors would make the outfit fitting for a job like a Bridge Assitant or Bridge Guard.)

> > I think maybe change the hat to a beret as there also common among command and have some level of associated authority in SS14
> 
> a beret in this shade of blue with the gold badge would be drippy
A beret is associated more with status and authority, and would therefore be better in atleast certain situations.


Maybe i should just add a beret and turtleneck as well, and allow players to pick the outfit combination they think fits whichever job they come up with the best.

The beret and turtleneck would work well for "irregular"/"custom"/non-roundstart heads as the beret and turtleneck are associated with authority (as you (plural) have mentioned).
I think the additions in this PR, the command jumpskirt, jumpsuit and cap, work better for command-related jobs that would impose less authority (Bridge Assitant, Bridge Guard, Command Intern, and perhaps even a Chief Service Worker or Head of Greytide).


(I am sorry for repeating myself, and other people, in this comment, i don't/didn't have time to write this perfectly.)
(I also won't have time to sprite or add a beret or turtleneck for atleast a few days.)

IProduceWidgets commented:
What about a suit style uniform? Something like you might expect a military officer to wear?

Catdere commented:
> The beret and turtleneck would work well for "irregular"/"custom"/non-roundstart heads as the beret and turtleneck are associated with authority (as you (plural) have mentioned). I think the additions in this PR, the command jumpskirt, jumpsuit and cap, work better for command-related jobs that would impose less authority (Bridge Assitant, Bridge Guard, Command Intern, and perhaps even a Chief Service Worker or Head of Greytide).

The turtlenecks are what centcom interns wear and a bridge officer type job that you are mention would be a bridge intern, As for having goofy heads in the 2 cases you mentioned, Head of Greytide wouldn't wear the uniform because they would want to keep wearing the grey uniform, The Head of Clowning would also be the same, and as for a Chief Service Worker... Thats already a job in game

GroggleG commented:
https://tgstation13.org/wiki/Bridge_Assistant
I feel would be the best reference, yellow and navy.

Emisse commented:
I agree with last two comments and referencing bridge assistant outfit from tg

GoldenCan commented:
I have now updated the sprites, and added a generic command beret.

The command cap can be used by command related jobs that are themselves not a "head" (jobs like tg's Bridge Assitant), whilst the command beret is intended for jobs that need to show off more authority (like the Head of Clowning).

Here they are in game:
![zCOMMANDCLOWNSANDSTUFF](https://github.com/user-attachments/assets/ee40f60f-ef86-45be-b184-b1caadee1ac5)
Shown left: A clown wearing only the command beret, because there is no requirement to wear the entire command outfit. (Which is something which i had a hard time describing through text for some reason.)
Shown in the middle: A kind of bridge officer/bridge guard character. (Or perhaps the captain's personal bodyguard? The possibilities are endless!)
Shown right: A human wearing a command jumpsuit and command cap.

PS. Thanks for all the feedback :)

LordEclipse commented:
> I have now updated the sprites, and added a generic command beret.
> 
> The command cap can be used by command related jobs that are themselves not a "head" (jobs like tg's Bridge Assitant), whilst the command beret is intended for jobs that need to show off more authority (like the Head of Clowning).
> 
> Here they are in game: ![zCOMMANDCLOWNSANDSTUFF](https://private-user-images.githubusercontent.com/175036423/368151302-ee40f60f-ef86-45be-b184-b1caadee1ac5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY1ODIzNzcsIm5iZiI6MTcyNjU4MjA3NywicGF0aCI6Ii8xNzUwMzY0MjMvMzY4MTUxMzAyLWVlNDBmNjBmLWVmODYtNDViZS1iMTg0LWIxY2FhZGVlMWFjNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxN1QxNDA3NTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MTAwNzdmZmM3ZGQ5NGFkNzdiYmM4NWY3NjNlNTllZDZlZmQxMDYwODQ2MmU1YzEwODIzNWZlZTcyMTI4M2ViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.6IKkKbXsIye5G5jg14ddFvUSYpGEZzwDxrI9m90qbfA) Shown left: A clown wearing only the command beret, because there is no requirement to wear the entire command outfit. (Which is something which i had a hard time describing through text for some reason.) Shown in the middle: A kind of bridge officer/bridge guard character. (Or perhaps the captain's personal bodyguard? The possibilities are endless!) Shown right: A human wearing a command jumpsuit and command cap.
> 
> PS. Thanks for all the feedback :)

That looks so much better! Awesome. Love it.

GoldenCan commented:
I can add a generic "Command Turtleneck" to my PR if it is requested, but have decided not to do so otherwise, as i consider it fluff.

I could also replace the generic "Command Jumpsuit" and "Command Jumpskirt" with a generic command turtleneck, but i have chosen not to as i consider the Command Jumpskirt/-suit to be useful in more in-game situations than a turtleneck.

GoldenCan commented:
Sprites for the command cap have been updated. See under "Media" above which has been updated.

UbaserB commented on Resources/Prototypes/Entities/Clothing/Head/soft.yml:
```diff
@@ -99,6 +99,22 @@
+  parent: ClothingHeadHeadHatBaseFlippable
+  id: ClothingHeadHatCommandsoft
+  name: command cap
+  description: "It's a baseball hat painted in Command colours."
```
why is it in quotation marks?

GoldenCan replied:
The quotation marks do not appear in game. Other descriptions in the files use quotation marks in the files as well.

UbaserB replied:
Huh… weird. I might make a github issue on that

lzk228 replied:
> why is it in quotation marks?

we use it when we have ' symbol in name or decs

SlamBamActionman commented on Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:
```diff
@@ -396,6 +396,17 @@
+  parent: [ClothingUniformBase, BaseCommandContraband]
+  id: ClothingUniformJumpsuitCommandGeneric
+  name: command jumpsuit
+  description: A generic command colored jumpsuit not associated with any particular department.
```
```suggestion
  description: A generic Command-colored jumpsuit not associated with any particular department.
```

SlamBamActionman commented on Resources/Prototypes/Entities/Clothing/Head/hats.yml:
```diff
@@ -192,6 +192,22 @@
+  parent: ClothingHeadBase
+  id: ClothingHeadHatBeretCommand
+  name: command beret
+  description: A beret with a command insignia emblazoned on it. It has an aura of authority.
```
```suggestion
  description: A beret with a Command insignia emblazoned on it. It has an aura of authority.
```

SlamBamActionman commented on Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:
```diff
@@ -221,6 +221,17 @@
+  parent: [ClothingUniformSkirtBase, BaseCommandContraband]
+  id: ClothingUniformJumpskirtCommandGeneric
+  name: command jumpskirt
+  description: A generic command colored jumpsuit not associated with any particular department.
```
```suggestion
  description: A generic Command-colored jumpsuit not associated with any particular department.
```

SlamBamActionman reviewed (approved):
Sure, seems fun. Just make some small text changes so that we're consistent with Command being capitalized and we're good.

Requires a final art lead approval to merge.

TheShuEd commented:
Please resolve merge conflicts, and if it get art approval i hit the merg button

GoldenCan commented:
> Please resolve merge conflicts, and if it get art approval i hit the merg button

Sure. I will see if i still remember how.
